### ISSUE TITLE
Request: Add "Shuffle on Sleep" function

### ISSUE THREAD

**Issue by sanjihlv** [TheLogicMaster/SDH-AnimationChanger#55]
Hello, just thought I'd jump in here to request a function that shuffles the boot and suspend videos on every sleep. I have a lot of suspend videos so it would be cool to see a different once each time, not know what you're gonna get. Seems like something people would want. Hopefully it's not something that's too difficult to implement. Anyways, thanks for reading.

**Comment by joshmedeski** [comment]
According to the SD Repo website:
https://steamdeckrepo.com/help

We could attempt to download the videos to that folder and see if the "Shuffle Startup Movie" feature can work as-is if the videos are in the right place.

**Comment by joshmedeski** [comment]
@TheLogicMaster what was the reason for downloading them to the `~/homebrew/animations` folder instead?

I'm guessing there were some limitations or you wanted to be able to choose SteamdeckOS. I'd be willing to experiment with it but I'd like to know if you have any insight on that decision.

**Comment by TheLogicMaster** [comment]
> @TheLogicMaster what was the reason for downloading them to the `~/homebrew/animations` folder instead?
> 
> I'm guessing there were some limitations or you wanted to be able to choose SteamdeckOS. I'd be willing to experiment with it but I'd like to know if you have any insight on that decision.

This plugin existed before the official customization settings, so they were stored separately and sym-linked into the override locations as needed. I never integrated the official one into the plugin because the plugin still offered more control and integrating would likely have been messy and broken with updates. Another reason is just how animations are stored, since it's by unique ID rather than filename, as you would want for using the official customization. An option to copy installed animations into the official location, or maybe download directly there, has been considered, but the mostly plugin UI work to support that just hasn't been done.

**Comment by joshmedeski** [comment]
> An option to copy installed animations into the official location, or maybe download directly there, has been considered, but the mostly plugin UI work to support that just hasn't been done.

I can write code that copies the installed animations onto the official location and add a "Copy to UI overwrites" button above "Reload Config".

My main concerns are:
1. Adding additional videos after you run the copy
2. How to prevent duplicates (if possible)

**Comment by TheLogicMaster** [comment]
Yeah, the concern for me was the amount of complexity added with regards to keeping track of copied animations and the UI required to manage it all. I was thinking of some klnd of "Download to override folder" button on the animation itself as a simple way to get that functionality without being responsible for anything in the folder. As for removing the symlink, selecting the default animation option in the plugin is meant to do that.
